ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -173,6 +168,13 @@ else
 		param reset
 	fi
 
+	#
+	# Start the tone_alarm driver.
+	# Needs to be started after the parameters are loaded (for CBRK_BUZZER).
+	# Note that this will still play the already published startup tone.
+	#
+	tone_alarm start
```

Comment:
This still seems a bit weird with the possibility of multiple tune_control calls before tone_alarm is possibly started, but I guess it's okay.